crates/sui/src/keytool.rs

```diff
@@ -60,6 +60,15 @@ mod keytool_tests;
 #[derive(Subcommand)]
 #[clap(rename_all = "kebab-case")]
 pub enum KeyToolCommand {
+    /// Update the old alias to the new alias.
+    /// If a new alias is not provided, a random one will be generated.
+    Alias {
+        /// Update the old alias name. Requires the old alias name to be the first value,
+        /// followed by the new alias name. If the new alias name is not provided, it
+        /// will be automatically generated.
+        #[clap(long, value_delimiter = ' ', num_args = 1..)]
+        update: Vec<String>,
```

Comment:
This doesn't really seem to be working in `clap`'s favour here.  How about making an `update-alias` command that accepts two positional arguments, with the second argument being optional?  This is also more inline with how other commands work (e.g. `sui client envs` with `sui client new-env` and `sui client update-env`).

Reply:
Much cleaner, thanks!